index.html

```diff
@@ -158,70 +165,74 @@ <h3 class="sm-text-2">A daily selection of privately personalized reads; no acco
+              <li><img src="./assets/html.svg" alt="" />HTML</li>
+              <li><img src="./assets/css.svg" alt="" />CSS</li>
+            </ul>
+            <div class="frameworks">
+              <h3 class="sm-text-2">Frameworks</h3>
+            </div>
+            <hr />
+            <div class="skills">
+              <h3 class="sm-text-2">Skills</h3>
+            </div>
+            <hr />
           </div>
-          <hr />
         </div>
-
-        <section class="contact-me">
+      </section>
+      <section class="bg-color-1">
+        <div class="contact-me">
           <div class="bg-image-2">
-            <div class="contact-container"></div>
-            <h2 class="lg-text-4">Contact me</h2>
-            <p class="sm-text-3">
-              If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d
-              love to help with it
-            </p>
+            <div class="contact-container">
+              <h2 class="lg-text-4">Contact me</h2>
+              <p class="sm-text-3">
+                If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                I’d love to help with it
+              </p>
 
-            <form action="https://formspree.io/f/xqknqlnl" method="POST" class="form">
-              <label for="fullname"></label>
-              <input type="text" id="fullname" name="fullname" placeholder="Full name" required maxlength="30" />
-              <label for="email"></label>
-              <input type="email" name="email" id="email" placeholder="name@domain.com" required />
-              <label for="feedback"></label>
-              <textarea
-                rows="5"
-                cols="30"
-                id="feedback"
-                name="feedback"
-                placeholder="Write your message here"
-                required
-                maxlength="500"
-              ></textarea>
-              <input type="submit" class="btn-1" value="Get in Touch" />
-            </form>
+              <form action="https://formspree.io/f/xqknqlnl" method="POST" class="form">
+                <label for="fullname"></label>
+                <input type="text" id="fullname" name="fullname" placeholder="Full name" required maxlength="30" />
+                <label for="email"></label>
+                <input type="email" name="email" id="email" placeholder="name@domain.com" required />
+                <label for="feedback"></label>
+                <textarea
+                  rows="5"
+                  cols="30"
+                  id="feedback"
+                  name="feedback"
+                  placeholder="Write your message here"
+                  required
+                  maxlength="500"
+                ></textarea>
+                <input type="submit" class="btn-1" value="Get in Touch" />
```

Comment:
- [ ] The `submit` or `Get in Touch` button should follow the button styling: Active, Hover, and Pressed. Kindly add these effects. This will make the website more interactive.